src/otx/core/metrics/fmeasure.py

```diff
@@ -644,6 +644,7 @@ class FMeasure(Metric):
     def __init__(
         self,
         num_classes: int,
+        best_confidence_threshold: float | None = None,
```

Comment:
Is there any problem if `best_confidence_threshold is None => vary_confidence_threshold is False` to merge them into one?

Reply:
In addition, is this not a scalar value but a 1d floating point vector in OTX 1.x?

Reply:
It is merged by [94b23d5](https://github.com/openvinotoolkit/training_extensions/pull/2996/commits/94b23d581f653f663a3d5a675f19f05818a38138)
And in OTX1.x self._best_confidence_threshold is ScoreMetric class and its value attribute is float